mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -2279,6 +2297,11 @@ func TestAccComputeInstance_spotVM_update(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckComputeInstanceExists(
 						t, "google_compute_instance.foobar", &instance),
+<% unless version == 'ga' -%>
+					testAccCheckComputeInstanceMaxRunDuration(&instance, expectedMaxRunDuration),
```

Comment:
Why do we want to check this explicitly? I think the test should already check this if the API returs the correct values?